pytorch_lightning/trainer/trainer.py

```diff
@@ -760,7 +761,10 @@ def _fit_impl(
 
         # TODO: ckpt_path only in v2.0
         ckpt_path = ckpt_path or self.resume_from_checkpoint
-        results = self._run(model, ckpt_path=ckpt_path)
+        self.fit_ckpt_path = self.__set_ckpt_path(
+            ckpt_path, model_provided=model, model_connected=self.lightning_module is not None
+        )
+        results = self._run(model, ckpt_path=self.fit_ckpt_path)
```

Comment:
One use of the `fit_ckpt_path` attribute is to enable callbacks to restore checkpoints within the context of fit() (e.g. [this tutorial](https://github.com/PyTorchLightning/lightning-tutorials/pull/115)) though I suspect it would be useful for other potential applications as well.